evap/results/exporters.py

```diff
@@ -285,12 +323,17 @@ def write_questionnaire(self, questionnaire, evaluations_with_results, contribut
 
     # pylint: disable=arguments-differ
     def export_impl(
-        self, semesters, selection_list, include_not_enough_voters=False, include_unpublished=False, contributor=None
-    ):
+        self,
+        semesters: Sequence[Semester],
+        selection_list,
+        include_not_enough_voters: bool = False,
+        include_unpublished: bool = False,
+        contributor: UserProfile | None = None,
+    ) -> None:
         # We want to throw early here, since workbook.save() will throw an IndexError otherwise.
         assert len(selection_list) > 0
 
-        for sheet_counter, (degrees, course_types) in enumerate(selection_list, 1):
+        for sheet_counter, (degrees, course_type_ids) in enumerate(selection_list, 1):
```

Comment:
Both seem to be `_ids` semantically from the usage below, at least if we annotated `filter_evaluations` and `write_headings_and_evaluation_info` below correctly.